nengo_loihi/simulator.py

```diff
@@ -563,6 +575,7 @@ def run_steps(self, steps):
                 )
             raise
 
+        self.timers["steps"] += timeit.default_timer() - time0
```

Comment:
The variable names `time0` can be renamed to something more descriptive e.g. `time_steps_start`